WalletWasabi/Hwi/Parsers/HwiParser.cs

```diff
@@ -336,10 +336,12 @@ public static string ToArgumentString(Network network, IEnumerable<HwiOption> op
 			options ??= Enumerable.Empty<HwiOption>();
 			var fullOptions = new List<HwiOption>(options);
 
-			if (network != Network.Main)
-			{
-				fullOptions.Insert(0, HwiOption.TestNet);
-			}
+			// TODO: Hackfix for making testnet usable on
```

Comment:
Instead of commenting this out, add that Trezor type to the conditions.

Reply:
Oh i wasnt aware that was possible haha :) Will do that